ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SyslogIntegrationForm.tsx

```diff
@@ -194,6 +199,36 @@ function SyslogIntegrationForm({
                                 isDisabled={!isEditable}
                             />
                         </FormLabelGroup>
+                        <FormLabelGroup
+                            label="Type"
+                            isRequired
+                            fieldId="messageFormat"
+                            touched={touched}
+                            errors={errors}
+                        >
+                            <ToggleGroup id="messageFormat" areAllGroupsDisabled={!isEditable}>
+                                        <ToggleGroupItem
+                                            // The HTML ID and custom CSS rule are required to make the shorter option similar in size to the longer option
+                                            // because PatternFly does not allow the inner width of the toggle button to be expanded easily
+                                            // (setting a min-witch on Toggle Item just adds space to the right of the outlined button)
+                                            id="CEF-option"
+                                            key='CEF'
+                                            text="CEF"
+                                            isSelected={values.syslog.messageFormat === 'CEF'}
+                                            onChange={() =>
+                                                setFieldValue('messageFormat', 'CEF')
+                                            }
+                                        />
+                                        <ToggleGroupItem
+                                            key='UNDEFINED'
+                                            text="CEF (legacy field order)"
```

Comment:
`CEF (Legacy)` is what I called in changelog. We can change in the parent PR if you wish.